Docker.DotNet/Properties/AssemblyInfo.cs

```diff
@@ -29,5 +29,5 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.16.0.0")]
+[assembly: AssemblyFileVersion("1.16.0.0")]
```

Comment:
I'm not sure it matters. I think I initially it was because of the version of the API it was supporting.

-----Original Message-----
From: "Ahmet Alp Balkan" <notifications@github.com>
Sent: ‎8/‎20/‎2015 1:11 PM
To: "ahmetalpbalkan/Docker.DotNet" <Docker.DotNet@noreply.github.com>
Cc: "Josh Garverick" <josh.garverick@gmail.com>
Subject: Re: [Docker.DotNet] Adding new NuSpec file with Microsoft.Bcl.Asyncdependency. (#19)

In Docker.DotNet/Properties/AssemblyInfo.cs:
> \ No newline at end of file
> +[assembly: AssemblyVersion("1.16.0.0")]
> +[assembly: AssemblyFileVersion("1.16.0.0")]
@jgarverick hey, I'm trying to understand why this change is needed.  finally had time to come back to this repo and release a new version.
—
Reply to this email directly or view it on GitHub.